engines/python/setup/djl_python/tests/neuron_test_scripts/test_neuron_tnx_rolling_batch.py

```diff
@@ -36,6 +38,21 @@
 }
 
 
+def download_model_from_s3(s3_url, model_name=None, download_dir=None):
```

Comment:
seems like model_name is required, let's remove the `=None`?

Reply:
model_name is not required if they gave download_dir.